boot/bootchain.go

```diff
@@ -62,13 +62,13 @@ func (b *bootChain) modelForSealing() *modelForSealing {
 }
 
 // TODO:UC20 add a doc comment when this is stabilized
```

Comment:
same here

Reply:
I will document this one. I do not think there will be much change here.